tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -2386,6 +2386,19 @@ public function userGetsAllTheSharesSharedWithHimUsingTheSharingApi(string $user
 		);
 	}
 
+  /**
+	 * @When user :user gets share :share shared with him using the sharing API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userGetsTheShareSharedWithHimUsingTheSharingApi(string $user, $share):void {
+		$user = $this->getActualUsername($user);
+    $shareId = $this->getLastPublicLinkShareId();
+    $this->getShareData($user, $shareId);
+	}
```

Comment:
weird indention

Reply:
I need help. I use visual studio code and I every time have problem with lint and indention: 
https://user-images.githubusercontent.com/84779829/214024185-c6738be4-e157-4eb6-aa1f-ee466e0b37be.mov

Reply:
https://user-images.githubusercontent.com/84779829/214025592-d5517da4-322f-4017-abb1-45eea68868aa.mov